themes/default/content/blog/pulumi-service-updates/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Improving the Pulumi Service Experience"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2022-07-14
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. This field is required or the build will fail the linter test. 
+# Max length is 160 characters.
+meta_desc: Pulumi Service, the easiest way to use Pulumi’s open-source universal infrastructure as code, just got better with a new and improved user experience.
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - anita-trimbur
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - pulumi-service
+    - user-experience
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+Earlier this year, Pulumi's user experience (UX) team sat down with users to understand how we can improve the Pulumi experience. It's important not only to the UX team, but across the company, that we listen to the real wants, needs, pains, and excitements of the people who use our product.
```

Comment:
Let's reframe the intro here to focus on the product and how we make it great.  Something closer to this narrative arc:
1. The Pulumi Service enables users and organizations to manage the breadth of cloud infrastructure they use across clouds, with features like X, Y, Z (core messaging from e.g. https://www.pulumi.com/product/pulumi-service/).
2. We initially launched the Pulumi Service console experience in 2018, and although the features and capabilities of the service have grown quickly over the last 4 years, the overall structure of the user experience has stayed largely consistent.
3. Today, we're launching a set of significant improvements to the overall user experience of the Pulumi Service console, making it easier to navigate around the console, surfacing even more important organizational insights into the dashboard and enabling quick access to the stacks you care most about with a new Favorites feature.

Specifically - I would lean less on the internal process we used and more on the product value.  I would also lean less on "here are all the problems with the existing product", and more on "here's what's even better now".

If we do want to have a follow-up blog post focused on UX design and development processes - that might be a good place to talk about some of these other themes - but I'd suggest they distract a bit from the product story as the lede here.